include/swift/AST/TensorFlow.h

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
Any reason not to remove the whole file?

Reply:
Ah, yes, I was being hacky when debugging the removal.